src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties

```diff
@@ -544,4 +544,26 @@ IMoleculeFragmenter.CycleFinderOption.relevant.tooltip = Relevant cycles of a gr
 IMoleculeFragmenter.CycleFinderOption.tripletShort.displayName = triplet short
 IMoleculeFragmenter.CycleFinderOption.tripletShort.tooltip =  The shortest cycle through each triple of vertices; allows to generate the envelope rings of some molecules (e.g. naphthalene) without generating all cycles
 IMoleculeFragmenter.CycleFinderOption.vertexShort.displayName = vertex short
-IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
\ No newline at end of file
+IMoleculeFragmenter.CycleFinderOption.vertexShort.tooltip = The shortest cycles through each vertex; linear independence is not checked; in practice similar to MCB
+##AlkylStructureFragmenter##
+AlkylStructureFragmenter.displayName = Alkyl Structure Fragmenter
```

Comment:
"Alkyl Fragmenter" is enough, this is displayed on the fragment button, so it must be short.